web/packages/test-helpers/src/generateBeefyCheckpoint.ts

```diff
@@ -47,23 +47,42 @@ async function generateBeefyCheckpoint() {
     const validatorSetId = await api.query.beefy.validatorSetId<ValidatorSetId>()
     const authorities = await api.query.beefy.authorities<BeefyId[]>()
 
+    console.log("validatorSetId:", validatorSetId)
+    console.log("authority length is:" + authorities.length)
+
     let addrs = []
+
     for (let i = 0; i < authorities.length; i++) {
-        let publicKey = publicKeyConvert(authorities[i], false).slice(1)
-        let publicKeyHashed = createKeccakHash("keccak256").update(Buffer.from(publicKey)).digest()
-        addrs.push(publicKeyHashed.slice(12))
+        console.log("index is:" + i + ",authority is:" + authorities[i])
+        try {
+            let publicKey = publicKeyConvert(authorities[i], false).slice(1)
+            let publicKeyHashed = createKeccakHash("keccak256")
+                .update(Buffer.from(publicKey))
+                .digest()
+            addrs.push(publicKeyHashed.slice(12))
+        } catch (err) {
+            console.log(err)
+        }
     }
 
-    const tree = createMerkleTree(addrs)
+    console.log("valid authority length is:" + addrs.length)
```

Comment:
This code doesn't look like its used for anything other than to print the length of the validator set. So maybe we can remove it?

Reply:
It's for debug purpose to print all invalid addresses, would prefer to keep it.